src/schemathesis/_hypothesis.py

```diff
@@ -80,3 +80,9 @@ def get_case_strategy(endpoint: Endpoint) -> st.SearchStrategy:
         body=from_schema(endpoint.body),
         form_data=from_schema(endpoint.form_data),
     )
+
+
+def register_string_format(name: str, strategy: st.SearchStrategy) -> None:
```

Comment:
Please, add runtime validation for the arguments - we need to avoid improper usage

Reply:
What about string for ‘name’?